articles/how-to-guides/create-an-application-template/create-an-application-template.md

```diff
@@ -2,3 +2,83 @@
 uid: how-to-guides.create-an-application-template
 ---
 # Create an Application Template
+
+Do you find yourself creating new applications in Intent Architect by choosing a very simple application template and customizing it by installing lots of modules?
+Working in a small team or organization can become daunting when someone else needs to create a new application and are not sure what the configuration settings should be.
+
+In this how-to we will explain how to setup your own team or organizational application template list which can run side-by-side to the official application template list and allows developers not only to create their own applications with ease, but they can easily customize it without having too much in-depth knowledge upon creation.
+
+In Intent Architect, choose to create a new application and select the `Intent Application Template Builder`.
+Give it the name of `MyApplicationTemplates`.
+
+![Application Template Builder Creation](images/my-app-templates-creation.png)
+
+When clicking on `Next` you will be presented with the following page. There is nothing to customize here so you can click on `Create` to begin your own application templates.
+
+![Application Template Builder Customization](images/my-app-templates-customize.png)
+
+Once the installation dialog is complete, click on `Close`.
+
+![Installation dialog](images/installation-dialog.png)
+
+Open the `App Templates` designer and click on `Create new pacakge`.
+
+![Create App Template Package](images/create-app-templates-package.png)
+
+If you find that the package is highlighted in red, click on it and complete the following fields:
+
+![Application Template Package Settings](images/app-template-pacakge-settings.png)
+
+| Name | Description |
+|-|-|
+| Version | You can give it a default value of `1.0.0`. **It needs to have 3 version components.** |
+| Display Name | Give it a display friendly name. Example: `My Standardized Application Templates` |
+| Supported Client Versions | Which versions of Intent Architect does this target? Click [here](xref:references.notations#version-ranges) to find out about the version scheme. Use the default of: `[3.0.0,4.0.0)` |
+
+Once this is specified, follow through with the following steps (as an example) on how to setup Groups, Components and Modules.
+
+<p><video style="max-width: 100%" muted="true" loop="true" autoplay="true" src="videos/create-group-components.mp4"></video></p>
+
+As per the video, create a Group called `Presentation` and a Component in it called `Angular` (give it the icon URL of `https://www.elemental.co.za/cms/resources/uploads/blog/89/05bd53609c4c.png`, you can click on the `...` button which will allow you to paste the image data in as well from the OS clipboard), then to finish it off, create a Module in the Component that is the same name as the actual Module that you wish to import into your application: `Intent.Angular`. Specify that Module to use version `3.0.0-beta.6`.
```

Comment:
I feel it's poor form of us to link directly to some website's hosted image, also, if they remove that image then this how-to breaks. Let's rather encourage them to copy/paste images. I think we should create an article on setting icons and then reference that here.

Also, @garethbaars, can we not just allow the component to have a default icon, or do we need to force people to choose one?